repos/system_upgrade/el7toel8/actors/localreposinhibit/actor.py

```diff
@@ -0,0 +1,83 @@
+from typing import List
+
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import (
+    CustomTargetRepository,
+    TargetRepositories,
+    UsedTargetRepositories,
+    UsedTargetRepository,
+)
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+
+
+class LocalReposInhibit(Actor):
+    """Inhibits the upgrade if local repositories were found.
+
+    Currently we're not supporting local repositories during the upgrade.
+    There is a workaround available, which described inside the remediation of
+    the report.
+
+    We're looking for a baseurl only inside TargetRepositories.custom_repos
+    because local repos might exists as only custom repository, due to the
+    fact that they might be specified only inside custom repo file of the leapp
+    or via --enablerepo flag. Both options puts the repo into
+    CustomTargetRepository.
+    """
+
+    name = "local_repos_inhibit"
+    consumes = (UsedTargetRepositories, TargetRepositories)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, TargetTransactionChecksPhaseTag)
+
+    def process(self):
+        used_target_repos = next(
+            self.consume(UsedTargetRepositories)
+        ).repos  # type: List[UsedTargetRepository]
```

Comment:
Is there a reason to not have it on one line? Like:
```suggestion
        used_target_repos = next(self.consume(UsedTargetRepositories)).repos  # type: List[UsedTargetRepository]
```
The line below is the same.

Reply:
I try to stick to 79, since, at first it is not violating 119 limits, second, it is much friendly for doing code reviews on mobile phones. Being a parent of two kids, this is a great advantage to me:)
Howe ver I know, sometimes the formatting becomes pretty ugly due to this constraint. This is an example:)